### PR TITLE
Extends README and adds service discovery helper methods to NetSuite class

### PR DESCRIPTION
**Pull request by jjorissen52**
My main goal with this PR was to make it a bit easier to get started with this library if you are unfamiliar with NetSuite's SOAP API (like me!). I ended up adding some helper methods that I found helpful while I was trying to learn the ropes.

This PR includes:

- Helper methods in `netsuite.client.NetSuite` to facilitate service discovery
- "Getting Started" information about programmatic usage and configuration to the README
- Description of how to use the service discovery helper methods added to README (thanks to @WesleyBatista for helpful snippet in https://github.com/jmagnusson/netsuite/pull/3#issuecomment-434866711)

### CODE REVIEW

File: netsuite/constants.py [hunk 1]
```diff
@@ -6,3 +6,150 @@
     os.path.expanduser('~/.config/netsuite.ini'),
 )
 DEFAULT_INI_SECTION: str = 'netsuite'
+
+NAMESPACES = [
```
[the remark this answers is not in the document]
Comment (jjorissen52) [reply netsuite/constants.py]: Yes, that list ended up being unnecessary. Should I do another commit to remove it or do you want to modify in the merge?

**Comment by jmagnusson** [comment]
Push another commit and I’ll properly review/merge when I have the time 😊

**Comment by thisjustin** [comment]
I can't comment on most of the code in part because we're just starting our journey into the NS jungle but huge +1 on the additional docs - we used these to get up and going with some initial tests of our integration.

**Comment by jjorissen52** [comment]
@jmagnusson Is it possible to give this a look soon? I plan to start using this package in one of my projects and I'd prefer to use it over my own fork.

**Comment by thosieucoi** [comment]
Hi @jjorissen52, would you please upgrade it to user version 2019_2_0?

**Comment by jmagnusson** [comment]
I'm going to close this because of all the changes that have happened. I might have time to take a look at this if you rebase against latest and move the documentation to the new dedicated docs.

**Pull request closed without merging**